README.md

```diff
@@ -1,2 +1,3 @@
 # JavaScriptLearning
 Contains practice codes I did while learning JavaScript.
```

Comment:
PR title is not apt, also there is a spelling mistake. Please make sure you don't repeat such things

Reply:
I'll keep this in mind and I will not repeat this.

Reply:
Please update the PR title